### PR TITLE
Users/dturcu/change recipe structure

### PR DESCRIPTION
**Pull request by turcudaria**
# Justification

#97 

# Implementation

add utensils and measure unit on ingredients

# Testing

http://localhost:8080/kitchen/swagger-ui/

# Checklist

- [ ] I tested changes to product code in product
- [ ] I considered updates to the README
- [ ] I considered updates to the error code dictionary

### CODE REVIEW

File: iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/Model/Request/RecipeIngredientRequest.java [hunk 1]
```diff
@@ -17,4 +17,7 @@
 
     @Min(value = 1, message = "The ingredient amount is not valid!")
     private int amount;
+
+    @NotBlank(message = "The um is not valid!")
+    private String um;
```
Comment (EnsyFane) [inline iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/Model/Request/RecipeIngredientRequest.java:22]: rename to measurement unit

File: iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/Model/Response/RecipeIngredientResponse.java [hunk 2]
```diff
@@ -10,4 +10,5 @@
 public class RecipeIngredientResponse {
     private String ingredientName;
     private int amount;
+    private String um;
```
Comment (EnsyFane) [inline iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/Model/Response/RecipeIngredientResponse.java:13]: measurementUnit

File: iChef-WebApi/src/main/java/com/kitchen/iChef/DTO/RecipeIngredientDTO.java [hunk 3]
```diff
@@ -10,4 +10,5 @@
 public class RecipeIngredientDTO {
     private String ingredientName;
     private int amount;
+    private String um;
```
Comment (EnsyFane) [inline iChef-WebApi/src/main/java/com/kitchen/iChef/DTO/RecipeIngredientDTO.java:13]: measurementUnit

File: iChef-WebApi/src/main/java/com/kitchen/iChef/Domain/RecipeIngredient.java [hunk 4]
```diff
@@ -26,6 +26,8 @@
 
     private int amount;
 
+    private String um;
```
Comment (EnsyFane) [inline iChef-WebApi/src/main/java/com/kitchen/iChef/Domain/RecipeIngredient.java:29]: measurementUnit

**Review by EnsyFane — APPROVED** [review]

**Review by Stefpark77 — APPROVED** [review]

File: iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/Model/Request/RecipeIngredientRequest.java [hunk 5]
```diff
@@ -17,4 +17,7 @@
 
     @Min(value = 1, message = "The ingredient amount is not valid!")
     private int amount;
+
+    @NotBlank(message = "The um is not valid!")
```
Comment (EnsyFane) [inline iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/Model/Request/RecipeIngredientRequest.java:21]: change this message to measurementUnit too.

**Review by roxanatrufelea — APPROVED** [review]

**Pull request merged**